motor_control/throttle/Button.h

```diff
@@ -0,0 +1,22 @@
+#ifndef BUTTON_THROTTLE
+#define BUTTON_THROTTLE
+
+#include "base_throttle.h"
+
+class Button: public Throttle{
```

Comment:
I think we should call this class `ButtonThrottle`, just because we do have other buttons in the car. The alternative would be to namespace these throttle classes so that they would be called like `throttle::Button` or `throttle::Variable`, but I think just sticking with the former approach is easiest.